docs/en/Community-Articles/2022-02-22-Integrating-MAUI-Client-via-using-OpenID-Connect/README.md

```diff
@@ -595,7 +595,7 @@ In this step we have to store access token & refresh token for future requests.
 
 ## Recap
 
-The purpose of this arcitle is connecting to ABP backend with access token and it's working properly.
+The purpose of this article is connecting to ABP backend with access token and it's working properly.
 
-I'm planning to integrate HttpApi.Client library of backend project instead of making requests manually as a second part of this article. I'll get inspired by [hikalkan/maui-abp-playing](https://github.com/hikalkan/maui-abp-playing) repo to achive that.
+I'm planning to integrate HttpApi.Client library of backend project instead of making requests manually as a second part of this article. I'll get inspired by [hikalkan/maui-abp-playing](https://github.com/hikalkan/maui-abp-playing) repo to archive that.
```

Comment:
```suggestion
I'm planning to integrate HttpApi.Client library of backend project instead of making requests manually as a second part of this article. I'll get inspired by [hikalkan/maui-abp-playing](https://github.com/hikalkan/maui-abp-playing) repo to achieve that.
```

Fix suggested from comments